MobileSDK/sdk/src/main/java/org/lantern/mobilesdk/Lantern.java

```diff
@@ -66,7 +66,7 @@
             System.setProperty("http.proxyPort", port);
             System.setProperty("https.proxyHost", host);
             System.setProperty("https.proxyPort", port);
-            return result;
+            return new StartResult(result.getHTTPAddr(), result.getSOCKS5Addr());
```

Comment:
Oh hey, I can get around this by making the one Xamarin binding project for sdk.aar depend on the other for liblantern.aar.  Closing PR (sorry for the spam).